src/lib/modules/pipeline/index.js

```diff
@@ -209,14 +211,29 @@ class Pipeline {
       },
       function runWebpack(shouldNotRun, next) {
         if(shouldNotRun) return next();
-        self.logger.info(__(`running webpack with '${self.webpackConfigName}' config...`));
         const assets = Object.keys(self.assetFiles).filter(key => key.match(/\.js$/));
         if (!assets || !assets.length) {
           return next();
         }
+        let strAssets = '';
+        if(!self.useDashboard){
         assets.forEach(key => {
-          self.logger.info(__("writing file") + " " + (utils.joinPath(self.buildDir, key)).bold.dim);
+          strAssets += ('\n  ' + (utils.joinPath(self.buildDir, key)).bold.dim);
         });
+      }
+        const timer = new LongRunningProcessTimer(
+          self.logger,
+          'webpack',
+          '0',
+          `${'Pipeline:'.cyan} Webpacking dapp using '${self.webpackConfigName}' config...${strAssets}`,
```

Comment:
I would change "Webpacking" to "Building" or "Packing", just so we don't have to change the message if we change from webpack to something else

Reply:
Changed to "bundling"